geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -190,34 +350,37 @@ These commands are supported for Redis 5.
 
    - uptime_in_days
 
- - stats
+- stats
 
-    - total_commands_processed
+   - total_commands_processed
 
-    - instantaneous_ops_per_sec
+   - instantaneous_ops_per_sec
 
-    - total_net_input_bytes
+   - total_net_input_bytes
 
-    - instantaneous_input_kbps
+   - instantaneous_input_kbps
 
-    - total_connections_received
+   - total_connections_received
 
-    - keyspace_hits
+   - keyspace_hits
 
-    - keyspace_misses
+   - keyspace_misses
 
-    - evicted_keys (always returns 0)
+   - evicted_keys (always returns 0)
 
-    - rejected_connections (always returns 0)
+   - rejected_connections (always returns 0)
 
-**[3]**  SLOWLOG is implemented as a NoOp.
+**[5]**  SLOWLOG is implemented as a NoOp.
```

Comment:
SLOWLOG should probably not be documented, even if it is listed internally as a supported command, because it's not actually implemented. Listing it here is just likely to cause confusion.